acceptance/setup/pre_suite/80_add_dev_repo.rb

```diff
@@ -1,40 +1,8 @@
+install_opts = options.merge( { :dev_builds_repos => ["PC1"] })
+repo_config_dir = 'tmp/repo_configs'
 if (test_config[:install_type] == :package and not test_config[:skip_presuite_provisioning])
   databases.each do |database|
-    os = test_config[:os_families][database.name]
-    db_facts = facts(database.name)
-
-    sed_cmd = "sed 's/#{Regexp.escape(test_config[:package_build_host])}/#{Regexp.escape(test_config[:package_repo_host])}/'"
-
-    step "Add development repository on PuppetDB server" do
-      case os
-      when :debian
-        result = on database, "lsb_release -sc"
-        deb_flavor = result.stdout.chomp
-        apt_list_url = "#{test_config[:package_repo_url]}/repo_configs/deb/pl-puppetdb-#{test_config[:package_build_version]}-#{deb_flavor}.list"
-        apt_list_file_path = "/etc/apt/sources.list.d/puppetdb-prerelease.list"
-        on database, "curl \"#{apt_list_url}\" | #{sed_cmd} > #{apt_list_file_path}"
-        result = on database, "cat #{apt_list_file_path}"
-        Log.notify("APT LIST FILE CONTENTS:\n#{result.stdout}\n")
-        on database, "apt-get update"
-      when :redhat
-        el_version = db_facts["os"]["release"]["major"]
-        yum_repo_url = "#{test_config[:package_repo_url]}/repo_configs/rpm/pl-puppetdb-#{test_config[:package_build_version]}-el-#{el_version}-x86_64.repo"
-        yum_repo_file_path = "/etc/yum.repos.d/puppetlabs-prerelease.repo"
-        on database, "curl \"#{yum_repo_url}\" | #{sed_cmd} > #{yum_repo_file_path}"
-
-        result = on database, "cat #{yum_repo_file_path}"
-        Log.notify("Yum REPO DEFINITION:\n\n#{result.stdout}\n\n")
-      when :fedora
-        version = db_facts["os"]["release"]["full"]
-        yum_repo_url = "#{test_config[:package_repo_url]}/repo_configs/rpm/pl-puppetdb-#{test_config[:package_build_version]}-fedora-f#{version}-x86_64.repo"
-        yum_repo_file_path = "/etc/yum.repos.d/puppetlabs-prerelease.repo"
-        on database, "curl \"#{yum_repo_url}\" | #{sed_cmd} > #{yum_repo_file_path}"
-
-        result = on database, "cat #{yum_repo_file_path}"
-        Log.notify("Yum REPO DEFINITION:\n\n#{result.stdout}\n\n")
-      else
-        raise ArgumentError, "Unsupported OS '#{os}'"
-      end
-    end
+    install_puppetlabs_dev_repo database, 'puppetdb', test_config[:package_build_version],
```

Comment:
This isn't installing puppetdb but rather gathering the `.repo` file as part of a function set up in beaker. This is instead of doing the custom functions removed above. The `puppetdb` parameter is telling the function what component to get. This is the same way it is done in [puppet server](https://github.com/puppetlabs/puppetserver/blob/master/acceptance/suites/pre_suite/foss/30_install_dev_repos.rb#L9-L10). [Beaker docs](http://www.rubydoc.info/github/puppetlabs/beaker/Beaker/DSL/InstallUtils/FOSSUtils#install_puppetlabs_dev_repo-instance_method)